[PLATFORM-648] Larger Map modules

mondoreale opened the pull request:
This small update sets min/default sizes for all Map module types to match the design. This includes:
- Map Geo
- Map Image
- Heatmap

And anything else that uses `Map` component. Preview:

![x](https://user-images.githubusercontent.com/320066/57675470-69f2db80-7622-11e9-8d69-733844369f62.png)

cc @mattatgit @hpihkala 

_(Thank gods I built the resize constraint mechanism this way, yay!)_ 💃

mattatgit commented:
@mondoreale Looks pretty good. Don't suppose we have any control over the size of the map scale component ? Not a big deal or need to worry about now, but just wondering. Be nice to size it down a bit at some point if doing so is not a technical drama.

mondoreale commented:
I may have something for the zoom component. Will double-check before merging. 🤞

timoxley commented:
@mondoreale zoom looks hackable with some overriding CSS e.g. https://codepen.io/leemark/pen/dGgqLZ

mondoreale commented:
It seems that we could have custom zoom component, in general. It'd take some doing do. @timoxley's  link shows easier (and ugly) solution that works. I love it.

Here's how it's gon' 🐝

<img width="389" alt="Screenshot 2019-05-15 17 57 26" src="https://user-images.githubusercontent.com/320066/57790438-06a89c80-773b-11e9-868a-0930fc61f098.png">

Buttons are 24x24 squared with of 1px of solid `#DCDCDC` border. `+` and `-` are simply large `+` and `-` (would be hacky to alter that). Also added the shadow and removed the ugly outer border.

Will merge it like this (have to push it first).
